components/ILIAS/Authentication/classes/LoginPage/LoginPageLanguagesOverviewTable.php

```diff
@@ -39,6 +39,7 @@ class LoginPageLanguagesOverviewTable implements UI\Component\Table\DataRetrieva
 
     private static string $activate = 'activate';
     private static string $deactivate = 'deactivate';
+    private static string $edit = 'edit';
```

Comment:
Please use constants, not static properties.

Reply:
oh. sorry...  done